CHANGELOG.md

```diff
@@ -6,18 +6,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v12.0.0] 2022-06-13
+
 ### MIGRATION NOTES
 
 - The changes introduced in CUMULUS-2955 should result in removal of
   `files_granule_cumulus_id_index` from the `files` table (added in the v11.1.1
-  release).  The success of this operation is dependent on system ingest load
+  release).  The success of this operation is dependent on system ingest load.
 
   In rare cases where data-persistence deployment fails because the
   `postgres-db-migration` times out, it may be required to manually remove the
   index and then redeploy:
 
   ```text
-  DROP INDEX IF EXISTS postgres-db-migration;
+  > DROP INDEX IF EXISTS postgres-db-migration;
   DROP INDEX
```

Comment:
Is `DROP INDEX` the output of the command? Should we remove that for clarity?

Reply:
Excellent point

928e8c3b1